server/endpoints.test.ts

```diff
@@ -0,0 +1,92 @@
+import mongoose from 'mongoose';
+import { MongoMemoryServer } from 'mongodb-memory-server';
+import express from "express";
+
+import axios from 'axios';
+import { configure } from "./endpoints";
+import { Classes, Students, Subjects } from './dbDefs';
+
+let mongoServer: MongoMemoryServer;
+
+const testingPort = 17760;
+
+beforeAll(async () => {
```

Comment:
In the future we should probably set up a jest global setup and global teardown so we don't have to repeat this every test file.

Reply:
I agree in the long run. However, in the short run, all tests modify the DB, and they do so in different ways. For tests to be deterministic, this is a necessity. Again, maybe in the future we can be smarter about this.